pkg/tls/ca.go

```diff
@@ -29,14 +28,11 @@ type CA struct {
 	// more than this allowance in either direction.
 	clockSkewAllocance time.Duration
 
-	// The CA's private key.
-	privateKey *ecdsa.PrivateKey
+	// PrivateKey of the CA.
+	PrivateKey *ecdsa.PrivateKey
 
-	// The CA's certificate.
-	root *x509.Certificate
-
-	// The PEM X.509 encoding of `root`
-	rootPEM string
+	// Crt is the certificate and trust chain.
+	Crt Crt
```

Comment:
Related to my comments in codec.go, I'd change this to:

```suggestion
	Crt *Crt
```

Note that if you do this you'll also have to update `newCA` to start initializing this field.